### PR TITLE
Undefined index error introduced in #180

### PR DESCRIPTION
**Pull request by synbot**
## Undefined index error introduced in #180

Error occurs if your entity has no `id` property and you forget to change the $autoIncrementColumn value

### CODE REVIEW

**Comment by paulstatezny** [comment]
:pray:

**Comment by zpchavez** [comment]
:star: That shippable failure was just a timeout.

**Comment by paulstatezny** [comment]
:hand:

File: src/Synapse/Mapper/InserterTrait.php [hunk 1]
```diff
@@ -58,7 +64,7 @@ protected function insertRow(AbstractEntity $entity, array $values)
 
         $result = $statement->execute();
 
-        if ($this->autoIncrementColumn && ! $values[$this->autoIncrementColumn]) {
+        if ($this->autoIncrementColumn && ! Arr::get($values, $this->autoIncrementColumn)) {
```
Comment (paulstatezny) [inline src/Synapse/Mapper/InserterTrait.php]: Weren't we going to revert this change in lieu of the above Exception thrown in this file?

**Comment by paulstatezny** [comment]
:interrobang:

**Comment by zpchavez** [comment]
:star:

**Comment by paulstatezny** [comment]
:+1:

**Comment by paulstatezny** [comment]
:sunny: :arrow_up:
